src/test/e2e-emulated/contracts/structs.tact

```diff
@@ -245,6 +245,79 @@ struct OptionalFields {
     avatar: String?;
 }
 
+struct S1 {
+    a: Int;
+    b: Int;
+    c: Int;
+}
+
+fun destructuringTest1(): Int {
+    let s = S {
+        a: true,
+        b: 42
+    };
+
+    let S {a, b} = s;
+
+    return (a ? 1 : 0) + b;
+}
+
+fun destructuringTest2(): Int {
+    let s = S {
+        a: true,
+        b: 42
+    };
+
+    let S {_, b} = s;
```

Comment:
wait, `let S {_, b} = s` should be a type error: `_` means `_: _` and there is no field `_`

Reply:
Oh, so if the user only wants `b`, they should write `let S {b} = s;`, right?

Reply:
correct

Reply:
or just use `.b`